main.go

```diff
@@ -565,20 +575,6 @@ func main() {
 				if err := db.Create(newUser).Error; err != nil {
 					return fmt.Errorf("admin user creation failed: %w", err)
 				}
-
```

Comment:
please update the README that running `estuary setup` will no longer return an API key

Reply:
@neelvirdy after the readme is updated, I think this is ready to go

Reply:
updated